tokens/pepe.json

```diff
@@ -0,0 +1,11 @@
QuNSAxMDk1LjVjLjczOS0uMTMgMS40MDYuMDQgMiAuNS0uOTM3LjY5LTEuNjAzLjUzLTItLjV6TTc2NS41IDEwOTYuNWMtLjM5NyAxLjAzLTEuMDYzIDEuMTktMiAuNS41OTQtLjQ2IDEuMjYxLS42MyAyLS41ek05NDEuNSAxMDk3LjVjLjM5Ny0xLjAzIDEuMDYzLTEuMTkgMi0uNS0uNTk0LjQ2LTEuMjYxLjYzLTIgLjV6IiBmaWxsPSIjMDAwIi8+PHBhdGggb3BhY2l0eT0iLjAxIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTkzNy41IDEwOTguNWMuMzk3LTEuMDMgMS4wNjMtMS4xOSAyLS41LS41OTQuNDYtMS4yNjEuNjMtMiAuNXoiIGZpbGw9IiM1MzUzNTMiLz48cGF0aCBvcGFjaXR5PSIuMDEiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNNzY4LjUgMTA5OC41YzEuMzMzLjY3IDEuMzMzLjY3IDAgMHoiIGZpbGw9IiMwMDAiLz48cGF0aCBvcGFjaXR5PSIuODA4IiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTc3Mi41IDEwOTguNWM0LjMxNy0uNDIgOC4zMTcuMjUgMTIgMi00LjIwNS0uMjItOC4yMDUtLjg4LTEyLTJ6IiBmaWxsPSIjNUU1RjVCIi8+PHBhdGggb3BhY2l0eT0iLjAxIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTkzNC41IDEwOTguNWMuNzM5LS4xMyAxLjQwNi4wNCAyIC41LS45MzcuNjktMS42MDMuNTMtMi0uNXoiIGZpbGw9IiM1MzUzNTMiLz48cGF0aCBvcGFjaXR5PSIuMDEiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNOTI5LjUgMTA5OS41YTguMzk0IDguMzk0IDAgMCAxIDQgLjVjLTEuNTY2Ljc2LTIuODk5LjYtNC0uNXpNNzc0LjUgMTEwMS41YzEuMTAxLTEuMSAyLjQzNC0xLjI2IDQtLjVhOC4zOTQgOC4zOTQgMCAwIDEtNCAuNXoiIGZpbGw9IiMwMDAiLz48cGF0aCBvcGFjaXR5PSIuMDEiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNOTI2LjUgMTEwMS41Yy4zOTctMS4wMyAxLjA2My0xLjE5IDItLjUtLjU5NC40Ni0xLjI2MS42My0yIC41eiIgZmlsbD0iIzgzN0Y1RSIvPjxwYXRoIG9wYWNpdHk9Ii4wMSIgZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik03ODUuNSAxMTAxLjVjLTEuMTAxIDEuMS0yLjQzNCAxLjI2LTQgLjVhOC4zOTQgOC4zOTQgMCAwIDEgNC0uNXoiIGZpbGw9IiMxQzFDMUMiLz48cGF0aCBvcGFjaXR5PSIuMDEiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNOTIxLjUgMTEwMS41YTQuOTQ5IDQuOTQ5IDAgMCAxIDMgLjVjLTEuMjQ5Ljc0LTIuMjQ5LjU3LTMtLjV6IiBmaWxsPSIjMDAwIi8+PHBhdGggb3BhY2l0eT0iLjYiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNOTE1LjUgMTEwMi41djFjLTEuOTE1LS4yOC0zLjU4MS4wNS01IDFoLTVjMy4wNDQtMS41MSA2LjM3OC0yLjE4IDEwLTJ6IiBmaWxsPSIjNzE3MjcxIi8+PHBhdGggb3BhY2l0eT0iLjAxIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTkxNS41IDExMDMuNWMuNzM5LS4xMyAxLjQwNi4wNCAyIC41LS45MzcuNjktMS42MDMuNTMtMi0uNXoiIGZpbGw9IiM1MzUzNTMiLz48cGF0aCBvcGFjaXR5PSIuMDEiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNODA4LjUgMTEwNS41Yy4zOTctMS4wMyAxLjA2My0xLjE5IDItLjUtLjU5NC40Ni0xLjI2MS42My0yIC41eiIgZmlsbD0iIzJCMkIyQiIvPjxwYXRoIG9wYWNpdHk9Ii4wMSIgZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik05MTAuNSAxMTA0LjVhOC4zOTQgOC4zOTQgMCAwIDEgNCAuNWMtMS41NjYuNzYtMi44OTkuNi00LS41ek04MTYuNSAxMTA1LjVjMS4zMzMuNjcgMS4zMzMuNjcgMCAwek05MDYuNSAxMTA2LjVjLjc1MS0xLjA3IDEuNzUxLTEuMjQgMy0uNS0uOTQ4LjQ4LTEuOTQ4LjY1LTMgLjV6TTg0OS41IDExMDYuNWMtLjM5NyAxLjAzLTEuMDYzIDEuMTktMiAuNS41OTQtLjQ2IDEuMjYxLS42MyAyLS41eiIgZmlsbD0iIzAwMCIvPjxwYXRoIG9wYWNpdHk9Ii4wMSIgZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik05MDIuNSAxMTA3LjVjLjM5Ny0xLjAzIDEuMDYzLTEuMTkgMi0uNS0uNTk0LjQ2LTEuMjYxLjYzLTIgLjV6IiBmaWxsPSIjNTM1MzUzIi8+PC9zdmc+",
```

Comment:
The icon is too big. Please make sure the icon is <10 KB.

Reply:
@codingshot could you fix this please?